Backend/test/mood.test.ts

```diff
@@ -18,23 +18,20 @@ describe("moodPromptFunction", () => {
     vi.clearAllMocks();
   });
 
-  it("should return a positive mood when AI returns a positive mood", () => {
+  it("should return a positive mood when AI returns a positive mood", async () => {
```

Comment:
Now that my function is merged I think it will be good to add a test to see that you can generate a correct mood based on the input. (Without mocking Gemini)

Reply:
Same in suggestion